TSC-Charter.md

```diff
@@ -176,10 +187,43 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
```

Comment:
I feel like one member wanting it to be confidential should be enough, not depending on the chair.